core/templates/base-components/loading-message.directive.html

```diff
@@ -0,0 +1,8 @@
+<div ng-show="$root.loadingMessage" class="oppia-loading-fullpage">
+  <div class="oppia-align-center">
+    <span translate="<[$root.loadingMessage]>"></span>
+    <span class="oppia-loading-dot-one">.</span>
+    <span class="oppia-loading-dot-two">.</span>
+    <span class="oppia-loading-dot-three">.</span>
+  </div>
+</div>
```

Comment:
Add newline after this one (please check the "Files Changed" tab each time after you push, to catch things like this).

Reply:
Done!